src/plugins/intel_gpu/src/graph/network.cpp

```diff
@@ -1034,10 +1034,12 @@ void network::execute_impl(const std::vector<event::ptr>& events) {
         }
 
         execute_primitive(inst, events);
-        executed_prims++;
+        if (!inst->get_node().get_selected_impl()->is_cpu())
+            executed_prims++;
```

Comment:
Does it really improve host overheads? I'm curious, because during tuning this flush period I tried the same approach, and it showed worse results than unconditional increment. I don't have detailed comparison related to counting of gpu impls only vs any impl, but IIRC it was visible on bert-small-uncased-whole-word-masking-squad-0001-fp16 model

Reply:
Thanks for sharing your experience. Logically I cannot understand why cpu impl should affect the flushing efficiency... Let me check with bert-small-uncased model.

Reply:
But I decided not to investigate for this more, because this part did not show the major perf impact. => To remove